TSC-Charter.md

```diff
@@ -50,43 +67,54 @@ to continue attending TSC meetings as an observer.
 
 ## Section 5. Responsibilities of the TSC.  
 
-The TSC is responsible for all technical development within the Planetary Software Organization,
-including:
-
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Maintaining a list of affiliate projects
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and coding standards.
+The TSC exercises autonomy in setting up and maintaining procedures and policies,
+as well as management and administrative structures as it deems appropriate for the
+maintenance and operation of these projects and resources.
+
+The TSC is responsible for all technical development within the
+Planetary Software Organization, including:
+
+* Technical direction
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+* Creating new repositories and projects under the 
+  [_planetarysoftware_ GitHub organization](https://github.com/planetarysoftware)
+  as required
+* Maintaining a list of projects and working groups
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Maintaining the list of additional Collaborators
+* Setting and maintaining standards covering contributions of code,
+  documentation and other materials
 * Mediating technical conflicts between Collaborators or Planetary Software Organization
-projects.
+projects
```

Comment:
so, again I'm confused if we have more projects than TLPs?

Reply:
maybe we could link to section 6 here, as it's explained there?

Reply:
I think it is possible, in the grand scheme of the universe that we may have projects that are not 'Top-Level Projects.'  If not, then we should change our language again, because then we'd only have 'projects' and there would be no reason to decorate the term with 'top-level.'

The use of 'project' in this sentence (and other uses in this document not decorated by 'top-level') were intentionally written to be broad so we don't have to constantly word-smith the Charter.  You generally want a charter or society bylaws or whatever to be reasonably general.

Section 6 is the very next section, and I don't think referring to it particularly clarifies the role of the TSC to mediate conflicts between its sub-entities, which is what this bullet is really about, whatever shape those sub-entities take.

Reply:
good point, being general!